docs/content/Deep-Storage.md

```diff
@@ -49,21 +49,6 @@ If you are using the Hadoop indexer, set your output directory to be a location
 
 ## Community Contributed Deep Stores
 
-### Azure
```

Comment:
while we're at it, can we fix the indentation level in the section you left below? (should be `###`, instead of `##`)